### DISCUSSION TITLE
[MinimalLib] Can I read a multiple molecule in sdf file ?

### DISCUSSION THREAD

**Discussion by seongsangcho-hits** [rdkit/rdkit#5370] [Q&A]
I can't find multiple read func in minimalLib so I converted a file(has multiple molecule) to string array and it split using `$$$$` end keyword then looping and reading a single string item. (Maybe It is not normal way.)

**Comment by greglandrum** [comment]
@seongsangcho-hits : at the moment there's no way to directly read in mulitiple molecules and your approach of splitting on `$$$$` tokens is what I would do.
